### PR TITLE
Feature csv

### PR DESCRIPTION
**Pull request by siddvish21**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #Update your aternos credentials here below
 
-ATERNOS_USERNAME=''
-ATERNOS_PASSWORD=''
+ATERNOS_USERNAME='siddharthalternos'
+ATERNOS_PASSWORD='siddharthplayingminecraft'
```
Comment (rsrini7) [inline .env]: again password added. You are using git add . always that is causing the problem.
Solve using this "A note about .env file security" section from https://www.twilio.com/blog/environment-variables-python

File: main.py [hunk 2]
```diff
@@ -39,27 +47,31 @@ def check_player():
...
-      wb = openpyxl.load_workbook('server-player-list.xlsx')
-      sheet = wb.active
-      sheet["A1"] = "Player name"
-      sheet["B1"] = "Leave/Join"
-      sheet["C1"]="Time"
+      data = list(zip(playername, leave_join, time))  
       for row in data:
         sheet.append(row)
-      wb.save('server1.xlsx')
+      wb.save('server-player-list.xlsx')
```
Comment (rsrini7) [inline main.py]: use constant to keep the hardcorded values and dont repeat

File: main.py [hunk 3]
```diff
@@ -39,27 +47,31 @@ def check_player():
...
+      with open(filename, 'w', newline="") as file:
+        csvwriter = csv.writer(file)
```
Comment (rsrini7) [inline main.py]: header will be repeated with this logic

**Review by rsrini7 — COMMENTED** [review]
if and elif blocks have duplicate code. try to minimize that by using functions

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+.env
```
Comment (rsrini7) [inline .gitignore:1]: good

File: main.py [hunk 5]
```diff
@@ -16,6 +16,26 @@ def currentTime():
...
+def data_to_sheet(data):
+  wb = openpyxl.load_workbook('server-player-list.xlsx')
+  sheet = wb.active
+  sheet["A1"] = "Player name"
+  sheet["B1"] = "Leave/Join"
+  sheet["C1"]="Time"
+
+  for row in data:
+    sheet.append(row)
+    wb.save('server-player-list.xlsx')
```
Comment (rsrini7) [inline main.py:28]: use constants instead of repeating multiple times of same hard-coded file name

File: main.py [hunk 6]
```diff
@@ -16,6 +16,26 @@ def currentTime():
...
+    
+
+def data_to_csv(data):
+  filename = 'aternos-player-list.csv'
+  header=['Player Name','Leave/Join','Time']
+  csvwriter.writerow(header)
```
Comment (rsrini7) [inline main.py:34]: this line will be added new header each time when you call data_to_csv function

**Review by rsrini7 — APPROVED** [review]
very good improvements

File: .env (hunk 1 above)
Comment (siddvish21) [reply .env]: Oh ok got it

**Comment by rsrini7** [comment]
You feature branch have old commits too 2b5dfcafabe96c89977123e7e82ef3b493173f06
You can learn how to revert old commits completely before pushing to github

**Pull request merged**